nemo_text_processing/text_normalization/normalize.py

```diff
@@ -254,8 +262,8 @@ def normalize(
         Returns: spoken form
         """
         if len(text.split()) > 500:
-            logging.warning(
-                "Your input is too long, and this could take a long time to normalize."
+            print(
+                "WARNING! Your input is too long and could take a long time to normalize."
```

Comment:
Do we want to switch from logging to warning?

Reply:
Going to merge this now. Please change this to logging in a future PR.